### PR TITLE
remove references to $gray-* outside _variables.scss

### PR DESCRIPTION
**Pull request by christopherjbaker**
I was playing around with some customizations and noticed that there were just a few references to $gray-\* variables outside the _variables.css file. As nearly all references are in _variables.scss, this seemed out of place.

There were 10 total references:
- 5 solved by the addition of 4 new variables
- 5 solved by the use of 2 existing variables

### CODE REVIEW

**Comment by cvrebert** [comment]
CC: @mdo

File: scss/utilities/_background.scss [hunk 1]
```diff
@@ -5,12 +5,12 @@
 // Inverse
 // TODO: redo this as a proper class
 .bg-inverse {
-  color: $gray-lighter;
-  background-color: $gray-dark;
+  color: $body-bg;
```
Comment (grizzm0) [inline scss/utilities/_background.scss]: $body-bg is #fff while $gray-lighter is #eceeef

**Comment by HMUDesign** [comment]
I added two new variables, `$inverse-bg` and `$inverse-color`, not only maintaining the original colors, but allowing a bit more customization.

**Comment by XhmikosR** [comment]
This needs a rebase and @mdo's feedback.

**Comment by mdo** [comment]
With a rebase, I think we can get this merged.

**Comment by mdo** [comment]
Thanks!

**Pull request merged**